src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -80,6 +80,11 @@ public function createAction(Request $request): Response
 
             return $this->redirectToRoute('admin_product_new');
         }
+        if ($this->get('prestashop.adapter.multistore_feature')->isUsed()) {
+            $this->addFlashMessageNotMultistoreCompatible();
```

Comment:
I'm not sure you need a flash message since you could display it in the content of the template you created But both work anyway

Reply:
I'll do that, it'll deal with both edit and creation 👍